python/processors/Stop0l_postproc.py

```diff
@@ -62,76 +65,96 @@
                     "redoJEC": False,
+                    "taggerWD": "TopTaggerCfg-DeepResolved_DeepCSV_GR_nanoAOD_2016_v1.0.2",
                    },
         "2016E" : { "JEC": "Summer16_07Aug2017EF_V11_DATA",
                     "redoJEC": False,
+                    "taggerWD": "TopTaggerCfg-DeepResolved_DeepCSV_GR_nanoAOD_2016_v1.0.2",
                    },
         "2016F" : { "JEC": "Summer16_07Aug2017EF_V11_DATA",
                     "redoJEC": False,
+                    "taggerWD": "TopTaggerCfg-DeepResolved_DeepCSV_GR_nanoAOD_2016_v1.0.2",
                    },
         "2016G" : { "JEC": "Summer16_07Aug2017GH_V11_DATA",
                     "redoJEC": False,
+                    "taggerWD": "TopTaggerCfg-DeepResolved_DeepCSV_GR_nanoAOD_2016_v1.0.2",
                    },
         "2016H" : { "JEC": "Summer16_07Aug2017GH_V11_DATA",
                     "redoJEC": False,
+                    "taggerWD": "TopTaggerCfg-DeepResolved_DeepCSV_GR_nanoAOD_2016_v1.0.2",
                    },
 
         "2017B" : { "JEC": "Fall17_17Nov2017B_V32_DATA",
                     "redoJEC": False,
+                    "taggerWD": "TopTaggerCfg-DeepResolved_DeepCSV_GR_nanoAOD_2017_v1.0.2",
                    },
         "2017C" : { "JEC": "Fall17_17Nov2017C_V32_DATA",
                     "redoJEC": False,
+                    "taggerWD": "TopTaggerCfg-DeepResolved_DeepCSV_GR_nanoAOD_2017_v1.0.2",
                    },
         "2017D" : { "JEC": "Fall17_17Nov2017DE_V32_DATA",
                     "redoJEC": False,
+                    "taggerWD": "TopTaggerCfg-DeepResolved_DeepCSV_GR_nanoAOD_2017_v1.0.2",
                    },
         "2017E" : { "JEC": "Fall17_17Nov2017DE_V32_DATA",
                     "redoJEC": False,
+                    "taggerWD": "TopTaggerCfg-DeepResolved_DeepCSV_GR_nanoAOD_2017_v1.0.2",
                    },
         "2017F" : { "JEC": "Fall17_17Nov2017F_V32_DATA",
                     "redoJEC": False,
+                    "taggerWD": "TopTaggerCfg-DeepResolved_DeepCSV_GR_nanoAOD_2017_v1.0.2",
                    },
 
         "2018A" : { "JEC": "Autumn18_RunA_V8_DATA",
                     "redoJEC": True,
+                    "taggerWD": "TopTaggerCfg-DeepResolved_DeepCSV_GR_nanoAOD_2018_v1.0.2",
                    },
         "2018B" : { "JEC": "Autumn18_RunB_V8_DATA",
                     "redoJEC": True,
+                    "taggerWD": "TopTaggerCfg-DeepResolved_DeepCSV_GR_nanoAOD_2018_v1.0.2",
                    },
         "2018C" : { "JEC": "Autumn18_RunC_V8_DATA",
                     "redoJEC": True,
+                    "taggerWD": "TopTaggerCfg-DeepResolved_DeepCSV_GR_nanoAOD_2018_v1.0.2",
                    },
         "2018D" : { "JEC": "Autumn18_RunD_V8_DATA",
                     "redoJEC": True,
+                    "taggerWD": "TopTaggerCfg-DeepResolved_DeepCSV_GR_nanoAOD_2018_v1.0.2",
                    },
             }
 }
 
-DeepResovledDiscCut = 0.6
```

Comment:
Why are we changing the cut?

Reply:
The final analysis cut is at 0.92, so this just cuts out a few unused tops to offset the space added by the systematics .